gems/pending/util/miq_apache/miq_apache.rb

```diff
@@ -206,7 +206,7 @@ def self.create_balancer_config(opts = {})
     def self.create_redirects_config(opts = {})
       opts[:redirects].to_miq_a.each_with_object("") do |redirect, content|
         if redirect == "/"
-          content << "RewriteRule ^/self_service(?!/(assets|images|img|styles|js|fonts)) /self_service/index.html [L]\n"
+          content << "RewriteRule ^/self_service(?!/(assets|images|img|styles|js|fonts|bower_components|gettext)) /self_service/index.html [L]\n"
```

Comment:
Not for today, just something to keep in mind since I don't understand why apache needs to know about this long list of things that the self service UI should be concerned with.